tools/lit-stories/.storybook-shared/config.ts

```diff
@@ -0,0 +1,52 @@
+// Required notice: Copyright (c) 2024, Will Shown <ch-ui@willshown.com>
+
+import { type StorybookConfig } from '@storybook/web-components-vite';
```

Comment:
i have a PR that renames tools/stories/.storybook-shared to just .storybook, which i think is the standard.

Reply:
Right, I intentionally named this something else so it could not be confused with a “standard” configuration, since it is not a complete configuration and does not work on its own (iirc).